### PR TITLE
Release 2019.02.0

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:

- Weekly release change log, event type updates, and documentation updates to go with the release.

### Resolves:
* [OKTA-209158](https://oktainc.atlassian.net/browse/OKTA-209158)

### CODE REVIEW

**Comment by mohitchugh-okta** [comment]
@manueltanzi-okta The core team doesn't own this repo. Was there someone else you were looking for?

**Comment by manueltanzi-okta** [comment]
Sorry @mohitchugh-okta , I meant to tag @motikagan-okta eheh

**Comment by motikagan-okta** [comment]
@manueltanzi-okta, Travis failures are now fixed.

File: _source/_use_cases/inline_hooks/api_am_hook/api_am_hook.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+### error
+
+When you return an error object, it should have the following structure:
+
+| Property     | Description                          | Data Type                   |
+|--------------|--------------------------------------|-----------------------------|
+| errorSummary | Human-readable summary of the error. | String                      |
+
+Returning an error object will cause Okta to return an OAuth 2.0 error to the requester of the token, with the value of `error` set to `server_error`, and the value of`error_description` set to the string you supplied in the `errorSummary` property of the `error` object you returned.
```
Comment (manueltanzi-okta) [inline _source/_use_cases/inline_hooks/api_am_hook/api_am_hook.md]: missing space between of and `error_description`

Comment (motikagan-okta) [reply _source/_use_cases/inline_hooks/api_am_hook/api_am_hook.md]: Thanks @manueltanzi-okta , now fixed.

**Pull request merged**